src/extras/curves/SplineCurve3.js

```diff
@@ -7,6 +7,7 @@ THREE.SplineCurve3 = THREE.Curve.create(
 
 	function ( points /* array of Vector3 */) {
 
+		THREE.warn( 'THREE.SplineCurve3 will be deprecated. Please use THREE.CatmullRomCurve3' );
```

Comment:
If you intent for users to get an error while using SplineCurve3 after this change, then yes. Would you like to remove the code now?